### PR TITLE
Fix Body Container bug

### PR DESCRIPTION
**Pull request by KirbyDaMaster**
## About The Pull Request

Fixed bug where body containers could not be closed when standing diagonally from them in some cases. Specifically when another solid object is adjacent to your character.

## Why It's Good For The Game

Bug go splat, fun go up :)

## Changelog

:cl:
fix: Enabled flag for morgue body containers to allow you to interact with them while standing diagonally with the hatch open.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
You want the changelog to describe behavior, rather then implementation details.
It's for the players, they should be able to understand it

**Comment by KirbyDaMaster** [comment]
> You want the changelog to describe behavior, rather then implementation details. It's for the players, they should be able to understand it

updated

File: code/game/objects/structures/morgue.dm [hunk 1]
```diff
@@ -399,7 +399,7 @@ GLOBAL_LIST_EMPTY(crematoriums)
 	name = "morgue tray"
 	desc = "Apply corpse before closing."
 	icon_state = "morguet"
-	pass_flags_self = PASSTABLE
+	pass_flags_self = LETPASSTHROW
```
Comment (LemonInTheDark) [inline code/game/objects/structures/morgue.dm:402]: why remove passtable? doesn't this mean things can't crawl under it (like aliens) now?

Comment (KirbyDaMaster) [reply code/game/objects/structures/morgue.dm]: Added passtable back, it was removed by mistake.

File: code/game/objects/structures/morgue.dm [hunk 2]
```diff
@@ -21,7 +21,7 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
 	density = TRUE
 	anchored = TRUE
 	max_integrity = 400
-
+	pass_flags_self = LETPASSTHROW
```
Comment (LemonInTheDark) [inline code/game/objects/structures/morgue.dm:24]: oh right and this needs | PASSSTRUCTURE doesn't it?

Comment (KirbyDaMaster) [reply code/game/objects/structures/morgue.dm]: amended

File: code/game/objects/structures/morgue.dm [hunk 3]
```diff
@@ -21,7 +21,7 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
 	density = TRUE
 	anchored = TRUE
 	max_integrity = 400
-
+	pass_flags_self = PASSSTRUCTURE | LETPASSTHROW
```
Comment (LemonInTheDark) [inline code/game/objects/structures/morgue.dm:24]: in order of the bitflags
```suggestion
	pass_flags_self = LETPASSTHROW | PASSSTRUCTURE
```

File: code/game/objects/structures/morgue.dm [hunk 4]
```diff
@@ -322,7 +322,7 @@ GLOBAL_LIST_EMPTY(crematoriums)
 	density = TRUE
 	var/obj/structure/bodycontainer/connected = null
 	anchored = TRUE
-	pass_flags_self = LETPASSTHROW
+	pass_flags_self = LETPASSTHROW | PASSTABLE
```
Comment (LemonInTheDark) [inline code/game/objects/structures/morgue.dm:325]:
```suggestion
	pass_flags_self = PASSTABLE | LETPASSTHROW
```

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Pull request merged**